tests/integration/security/authorization_test.go

```diff
@@ -1066,3 +1069,61 @@ func TestAuthorization_Path(t *testing.T) {
 			rbacUtil.RunRBACTest(t, cases)
 		})
 }
+
+// TestAuthorization_Audit tests that the AUDIT action does not impact allowing or denying a request
+func TestAuthorization_Audit(t *testing.T) {
```

Comment:
nit: prefer to reuse existing tests. This spins up a new namespace with 4 pods ,which takes ~25s to startup and ~15s to tear down. All of the pilot tests share the same pods and run various tests against them, and a result can run many tests in a couple minutes, vs the 30 min security test (the longest of all tests we have)